docs/handbook/systems-administration.md

```diff
@@ -28,109 +28,207 @@ activities and many more are referred to as system administration.
+
+OpenIndiana comes with several service accounts such as `webservd` for web servers,
+`pkg5srv` for `pkg(5)` server or `nobody`.
+`nobody` is a system account for services needing unprivileged user.
+However, the more services are ran under this account,
+the more privileged it becomes as it gains access to service processes and files.
+
+### Roles
+
+A role is a basic unit of Role-Based access control (RBAC) or set of privileges one can assume.
+A role can be thought of as a no-login account.
+It has most of the attributes of normal user account and is identified like normal user,
+but is not allowed to log into a system directly.
+One should login using regular account and use [su(1M)](https://illumos.org/man/1M/su) to assume the role.
+
+### Superuser account
+
+Every UNIX-like system has the superuser account, named root, used for system administrative tasks.
+This account has UID 0.
+
+Using this account for everyday usage like web browsing, email reading or movie watching is not recommended
+as root account can operate without any restrictions or limits and could cause serious damage to the system.
+
+If you created user during installation, root is created as role (not a regular account).
+This role is assigned to the user created during the installation.
+This means that you can't directly log into a system using the root account.
+One has to log in as the created user and switch to root role using [su(1M)](https://illumos.org/man/1M/su).
+
+However, if the user was not created during installation, then the root is created as regular account and
+is able to log in directly.
+
+#### SUperuser DO: sudo(1m)
+
+It is not always feasible for one user to perform all administrative
+tasks. It would be more flexible if some tasks could be performed by some, say,
+experienced users. To enable some users to carry out a command with _all_ root
+privileges, or to _do_ an administrative command `sudo(1M)` can be used.
 
-Hence, a mechanism was developed whereby users could be assigned a select number
-of privileges by the superuser. OpenIndia provides, in addition to these
-traditional  mechanisms, a richer means to perform these duties known as
-Role-Based Access Control.
+The `sudo` command, i.e., superuser do, permits a regular user to execute the specified
+set of commands with supperuser privileges without having to become the superuser.
 
-RBAC involves collecting a select number of privileges and bundling these
-together as a role. A user can then be assigned one, or several roles.
+If user was permitted to use some commands with superuser privileges via `sudo`
+he/she can execute them with elevated privileges simply starting command with `sudo`.
 
+To permit a user to use `sudo`, the superuser edits `/etc/sudoers`.
 
-### root: [su(1M)](https://illumos.org/man/1M/su)
+This can be done using special `visudo(1M)` tool.
+`visudo` tool edits `/etc/sudoers` file performing various syntax checks.
```

Comment:
Beside a local `/etc/sudoers`, the `sudo` program supports other databases (if compiled with corresponding options), such as certain data trees in LDAP along with other LDAP accounts integration. But again, maybe this docs improvement belongs in another iteration :)